ledger-core/virtual/integration/deduplication/constructor_test.go

```diff
@@ -259,6 +259,9 @@ func (test *DeduplicationDifferentPulsesCase) run(t *testing.T) {
 	if test.ExecuteShouldHaveError != "" {
 		foundError = test.Server.Journal.Wait(func(event debuglogger.UpdateEvent) bool {
 			if event.Data.Error != nil {
+				if _, ok := event.Data.Error.(throw.StackTraceHolder); !ok {
```

Comment:
why do you need this check?
DeepestStackTraceOf will then just return nil

Reply:
yes, fix it